api-specificatie/openapi.yaml

```diff
@@ -486,51 +483,38 @@ components:
           $ref: "#/components/schemas/Nummeraanduiding"
         woonplaatsen:
           $ref: "#/components/schemas/Woonplaats"
```

Comment:
Ontbreekt panden hier of is die bewust weggelaten? Ze zitten nl. wel in de links.

Reply:
Ben ik nog niet aan toegekomen. Panden was nog onderhanden werk.